README.md

```diff
@@ -207,15 +19,23 @@ The page, without clicking on "More", displays 20 previews of projects in the NY
 }
 ```
 
-This will probably be in a larger hash called `projects`. Sound good?
+These individual project hashes will be collected into a larger hash called `projects`. 
+
+## Fixtures
 
-### Some Considerations
+In the directory of this project you'll notice a folder called `fixtures`. Inside that folder you'll see a file, `kickstarter.html`. Open up that file and right click anywhere on the page. Select "open in browser" from the menu that appears. 
```

Comment:
it is briefly explained with the line "it is common to keep data the test suite will use in a `fixtures` directory"